### PR TITLE
fix: Update android docs

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: platform/formats/android_xml.mdx [hunk 1]
```diff
@@ -136,6 +137,61 @@ Since the `[` and `]` characters are not allowed in the key names in the Android
 there should be no conflicts if you use the Tolgee project only for the Android app. Otherwise, remember that keys named
 with the described syntax are converted to arrays.
 
+## Strings wrapped with CDATA
+When Tolgee imports the Android XML file, it extracts the strings from the CDATA sections. e.g.
+`<string><![CDATA[Hello]]></string>` is imported as `Hello`. Keys with imported translations with CDATA-wrapped values
+are marked with `_androidWrapWithCdata` [custom value](/platform/translation_keys/custom_values). When exporting,
+keys marked with `_androidWrapWithCdata` are wrapped with CDATA.
```
Comment (stepan662) [inline platform/formats/android_xml.mdx:144]: Does the underscore needs to be there?

**Review by stepan662 — APPROVED** [review]

File: platform/formats/android_xml.mdx (hunk 1 above)
Comment (JanCizmar) [reply platform/formats/android_xml.mdx:144]: Yep, all the properties which have a meaning for Tolgee are marked with the underscore. (see XLIFF, Flutter)

Comment (stepan662) [reply platform/formats/android_xml.mdx:144]: 😢

Comment (stepan662) [reply platform/formats/android_xml.mdx:144]: Then it's ok like this I guess.

**Pull request merged**